pixeltable/env.py

```diff
@@ -500,6 +503,10 @@ def _init_db(self, config: Config) -> None:
         assert self._db_url is not None
         assert self._db_name is not None
 
+    @retry(
+        stop=stop_after_attempt(3),  # Stop after 3 attempts
+        wait=wait_exponential_jitter(initial=0.2, max=1.0, jitter=0.2),  # Exponential backoff with jitter
+    )
```

Comment:
Maybe add a debug log message in the method body so the retries are apparent in the logs.